### PR TITLE
Merge dev to main

### PR DESCRIPTION
**Pull request by OscarFMdev**
# PR description: 📄 
In this branch we:
- Created a Single Page Application using React and Redux. :atom: 
- Created the components for Dragons, Rockets, and Missions. 🐲 
- Displayed the user selection on the My Profile page. 💾 
- Created tests for each of the pages. 🧪 

### Thank you for reviewing our code! 🙌🏽

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,241 @@
...
+-->
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+<div align="center">
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
Comment (descholar-ceo) [inline README.md:28]: - [ ] This Logo is broken

File: README.md [hunk 2]
```diff
@@ -1,46 +1,241 @@
...
+- **[The user can cancel reservations or joining missions]**
+- **[The user data is displayed in My Profile page.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
```
Comment (descholar-ceo) [inline README.md:101]: - [ ] Kindly add a working live demo link

**Review by descholar-ceo — APPROVED** [review]
Hi @OscarFMdev && @d4nQw3rty && @Solobarine 🤝 🤝 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽  Your Project looks great, keep up the good work ✔️ 
👍🏽 The PR is professional, professionally described ✔️ 
👍🏽 The PR has linters installed and noe errors ✔️ 
👍🏽 The PR is following the correctly git / github flows ✔️ 
👍🏽 The commit messages are meaningful ✔️ 
👍🏽 The readme file is professional with the acknowledgement section ✔️ 

## Optional suggestions
- [ ] Just before you merege address few comments I left in your readme file 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (OscarFMdev) [reply README.md:28]: Thank you for letting us know @descholar-ceo! 🙌🏽

File: README.md (hunk 2 above)
Comment (OscarFMdev) [reply README.md:101]: And also for letting us know this part! 😅

**Comment by OscarFMdev** [comment]
We'll fix both optional suggestions, thank you very much for helping us improve and for approving our project! Have a great day @descholar-ceo!

**Pull request merged**